### PR TITLE
Fixed changing name of a git user

### PR DESCRIPTION
**Pull request by nextsux**
I'm used to run as gitlab user (whole team has setup this user in remotes). There were few problems when GITLAB_USER is changed.

### CODE REVIEW

File: assets/runtime/env-defaults [hunk 1]
```diff
@@ -21,6 +21,7 @@ else
 fi
 
 ## SSH
+GITLAB_SSH_USER=${GITLAB_SSH_USER:-$GITLAB_USER}
```
Comment (solidnerd) [inline assets/runtime/env-defaults]: Default value for `$GITLAB_USER`is missing

File: assets/build/install.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ paxctl -Cm `which nodejs`
 rm -rf /etc/ssh/ssh_host_*_key /etc/ssh/ssh_host_*_key.pub
 
 # add ${GITLAB_USER} user
-adduser --disabled-login --gecos 'GitLab' ${GITLAB_USER}
+adduser --disabled-login --gecos 'GitLab' ${GITLAB_USER} --home ${GITLAB_HOME}
```
Comment (solidnerd) [inline assets/build/install.sh]: Why is setting the home directory needed ?

File: assets/runtime/env-defaults (hunk 1 above)
Comment (nextsux) [inline assets/runtime/env-defaults]: It's defined in Dockerfile

File: assets/build/install.sh (hunk 2 above)
Comment (nextsux) [inline assets/build/install.sh]: Without this adduser creates /home/$GITLAB_USER by default but everything else points to /home/$GITLAB_HOME

**Pull request closed without merging**